apps/chat/src/components/Header/BackToChatMobile.tsx

```diff
@@ -0,0 +1,33 @@
+import { IconArrowLeft } from '@tabler/icons-react';
+
+import { useTranslation } from 'next-i18next';
+import { useRouter } from 'next/router';
+
+import { Translation } from '@/src/types/translation';
+
+import Tooltip from '../Common/Tooltip';
+
+export const BackToChatMobile = () => {
+  const { t } = useTranslation(Translation.Header);
+
+  const router = useRouter();
+
+  return (
+    <Tooltip isTriggerClickable tooltip={t('Back to Chat')}>
+      <button
+        className="flex h-full items-center justify-center border-r border-tertiary px-2 md:px-3 lg:hidden"
```

Comment:
lg it's still tablet demension. Use xl: (desktop) or md: (for tablet and desktop)

Reply:
I thought to show the button until the side panel appears, but I checked again according to the design, there is no button on the tablet